docs/src/pages/docs/_meta.ts

```diff
@@ -0,0 +1,13 @@
+export default {
+  "-- 1": {
+    type: "separator",
+    title: "Docs",
+  },
```

Comment:
can be removed since you don't have several sections

Reply:
Thanks - removed was wondering what this was!